Workbooks/Virtual Machines - Network Dependencies/Connections Overview - Single VM/Connections Overview.workbook

```diff
@@ -2,34 +2,25 @@
   "version": "Notebook/1.0",
   "isLocked": true,
   "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "<h1>Connections Overview</h1>"
+      },
+      "conditionalVisibility": null
+    },
     {
       "type": 9,
       "content": {
         "version": "KqlParameterItem/1.0",
         "query": "",
-        "crossComponentResources": [],
+        "crossComponentResources": [
+          "{Computer}"
+        ],
         "parameters": [
           {
             "id": "e41c2177-932a-4c58-ba24-03ef070eb197",
             "version": "KqlParameterItem/1.0",
-            "name": "Workspace",
-            "type": 5,
-            "isRequired": true,
-            "value": "value::1",
-            "isHiddenWhenLocked": true,
-            "typeSettings": {
-              "resourceTypeFilter": {
-                "microsoft.operationalinsights/workspaces": true
-              },
-              "additionalResourceOptions": [
-                "value::1"
-              ]
-            },
-            "timeContextFromParameter": null
-          },
-          {
-            "id": "67b8a28c-f7f9-4903-883d-4a308a6ffe71",
-            "version": "KqlParameterItem/1.0",
             "name": "Computer",
```

Comment:
where is the Computer param shown in the mockup?

Reply:
The `Computer` param is hidden all the way at the top, it is a dropdown used to select the "default" Computer resource that is passed into this workbook
![image](https://user-images.githubusercontent.com/43890980/52749269-7e7cf480-2f9d-11e9-8adb-0e9bb91dcf78.png)